lib/aws_assume_role.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
```

Comment:
Can we also add:

```
shopt -s inherit_errexit
```

We are using subshells in this script, so we would want their failures to propagate